### PR TITLE
Add some log for delete task when it need query

### PR DESCRIPTION
**Pull request by aoiasd**
/related https://github.com/milvus-io/milvus/issues/25751

### CODE REVIEW

File: internal/proxy/task_delete.go [hunk 1]
```diff
@@ -300,28 +303,32 @@ func (dt *deleteTask) getStreamingQueryAndDelteFunc(stream msgstream.MsgStream,
...
-			log.Warn("query for delete return error", zap.Error(err))
+			log.Warn("query stream for delete create failed", zap.Error(err))
 			return err
 		}
 
 		for {
 			result, err := client.Recv()
 			if err != nil {
 				if err == io.EOF {
+					log.Info("query stream for delete finished", zap.Int64("msgID", dt.msgID), zap.Duration("duration", rc.ElapseSpan()))
```
Comment (yah01) [inline internal/proxy/task_delete.go:317]: Change it to debug level or maybe produce many logs

File: internal/proxy/task_delete.go [hunk 2]
```diff
@@ -262,13 +262,16 @@ func (dt *deleteTask) PostExecute(ctx context.Context) error {
...
 func (dt *deleteTask) getStreamingQueryAndDelteFunc(stream msgstream.MsgStream, plan *planpb.PlanNode) executeFunc {
 	return func(ctx context.Context, nodeID int64, qn types.QueryNodeClient, channelIDs ...string) error {
-		// outputField := translateOutputFields(, dt.schema, true)
-
 		partationIDs := []int64{}
 		if dt.partitionID != common.InvalidFieldID {
 			partationIDs = append(partationIDs, dt.partitionID)
 		}
 
+		log.Debug("start query for delete",
```
Comment (yah01) [inline internal/proxy/task_delete.go:270]: use
```golang
log := log.Ctx(ctx).With(
    zap.Int64("collectionID", dt.collectionID),
    ...
)
```
and remove these fields from line 360

**Review by yah01 — CHANGES_REQUESTED** [review]

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**